vendor/assets/stylesheets/dvl/components/filter_form.scss

```diff
@@ -1,81 +1,60 @@
 .filter_form {
-  display: inline-block;
-  position: relative;
-  .button,
-  .styled_select_wrapper {
-    margin-top: $rhythm;
-    display: block;
+  button {
+    @include button_color($lightestGray,$darkestGray);
+    border-left-color: transparent;
+    text-transform: uppercase;
+    letter-spacing: $letterspaceSmall;
   }
-  @media screen and (min-width: $lapWidth) {
-    .styled_select_wrapper,
-    .button {
-      margin-top: 0;
-      display: inline-block;
-      vertical-align: top;
+  // Search button is present but hidden
+  &.full {
```

Comment:
Are we using this modifier in our apps? If no, let's remove. If yes, let's add to docs.